openlibrary/plugins/importapi/code.py

```diff
@@ -181,7 +181,10 @@ def POST(self):
         if not can_write():
             return json.dumps({'success':False, 'error':'Permission Denied'})
 
-        i = web.input()
+        i = web.input()        
+
+        require_marc = False if i.get('require_marc') == 'false' else True
```

Comment:
Would this be more concise as `require_marc = not (i.get('require_marc') == 'false')`?

Reply:
great suggestion